notes/2019-08-01.md

```diff
@@ -0,0 +1,104 @@
+- Erik: We could absolutely create an issue to outline the complete effort
+- Lee: Yeah maybe we could find some folks for this.
+- Ivan: I think the graphiql team showed a good path to creating working groups, I added some links
+  - [graphql-wg#136](https://github.com/graphql/graphql-wg/issues/136)
+  - [graphiql Working Group Agendas](https://github.com/graphql/graphiql/tree/master/wg-agendas)
+- Erik: Definitely, I think we should reach out and organize some kind of meeting!
+- Ivan: Good to know that you cover all use cases. With graphiql, some people just wanted to use it as components or the full solution
+- Erik: I think the long term issues include things like adding validation support so there’s a lot of questions to be answered
+- Lee: Sounds like a great plan, I think we should strive to make libgraphqlparser speccompliant and get it back to the point it was supposed to be
+- Erik: Great so Lee you would grant me access to manage PRs
+- Lee: Excellent. GraphQL Cats!
+
+### [GraphQL Cats](https://github.com/graphql-cats/graphql-cats) ([Michael Staib](https://github.com/michaelstaib))
+
+- Michael: Yeah, we started to build a test generator for all the stuff half a year ago, but since then there hasn’t been much activity. Some of the rules are probably broken we found, if we want to have a project like GraphQL cats, we should add a rule to this framework once the spec changes. I don’t know if we want to follow this direction so storing it in yaml and generating specific tests might be good but if there’s some way to get this up to speed it would be great. We would love to help with this since we’re porting  tests from graphql-js back to C# at the moment, where the workflow is really manual. In each specific language we have a generator to create the tests. There’s a lot of changes in graphql-js where we have to look up what actually changed and if we need to update the tests. Any other thoughts on GraphQL cats? Has anybody else than GraphQL Java put in some effort? Could we get this up and running again?
+- Lee: Great questions. I don’t have any satisfying answers at the moment. If you are eager on takign this on, it seems like that’s the missing piece. I think there’s some ambiguity why we can’t make it run on all platforms at once, why would you build this on top of your engine when there’s only some tests but then why would you contribute to graphql cats? I think in the beginning we just have to force this through so whoever has the time and passion to pull this off might steer it. People that are new to the ecosystem might question if the tool will work for them would appreciate graphql cats.
+- Michael: The only blocker is that I’m not sure who owns graphql cats right now. If we were to put it into a place where people had to write tests upon changes to the spec. There are definitely rules we’d have to fix but overall it’s a great idea.
+- Lee: Does anybody know who else might have access to GraphQL cats?
```

Comment:
I own the GraphQL CATS org and repo. Would love to reboot this project and have it be a first class citizen along with the spec.

Reply:
Please let me know how I can help!

Reply:
@leebyron

Reply:
Awesome! Give me a moment to figure things out there. @joshprice let me know an email to contact you at and I'll fill you in on the details.